content/docs/user-guide/project-config/cli-reference.md

```diff
@@ -2,6 +2,1696 @@
+
+## Commands
+The `o3de` Python script contains the following commands, with further details in the following sections. 
+
+| Command | Description | 
+| - | - |
+| [`get-global-project`](#get-global-project) | Gets the global project that is registered to the engine. |
+| [`set-global-project`](#set-global-project) | Sets the specified project as the engine's global project. |
+| [`create-template`](#create-template) | Creates a template out of the specified source path. |
+| [`create-from-template`](#create-from-template) | Creates an instance from a generic template.  |
+| [`create-project`](#create-project) | Creates a new project. |
+| [`create-gem`](#create-gem) | Creates a new Gem. |
+| [`register`](#register) | Registers an O3DE object. |
+| [`register-show`](#register-show) | Shows the registered O3DE objects. |
+| [`get-registered`](#get-registered) | Shows the path to the registered O3DE object with the specified name. |
+| [`enable-gem`](#enable-gem) | Enables the Gem in your project. |
+| [`disable-gem`](#disable-gem) | Disables the Gem in your project. |
+| [`edit-engine-properties`](#edit-engine-properties) | Edits the engine's properties. |
+| [`edit-project-properties`](#edit-project-properties) | Edits the project's properties. |
+| [`edit-gem-properties`](#edit-gem-properties) | Edits the Gem's properties. |
+| [`sha256`](#sha256) | Creates a hash value for an O3DE object using the SHA-256 secure hash algorithm. |
+
+
+<!-------------------------------------------------------------->
+
+## `get-global-project`
+
+Gets the global project that is registered to the engine. By default, this is read from `<USER_PATH>/.o3de/Registry/bootstrap.setreg`. You can also specify a file path using `-i`.
+
+### Format
+
+```cmd
+get-global-project [-h] [-i INPUT_PATH]
+```
+
+### Usage
+
+Reads the global project from `<USER_PATH>/.o3de/Registry/bootstrap.setreg`.
+
+```cmd
+o3de.bat get-global-project
+```
+
+Reads the global project from a specified file.
+```cmd
+o3de.bat get-global-project -i <USER_PATH>\.o3de\Registry\my-custom.setreg
+```
+
+### Optional parameters
+
+- **`-h, --help`**
+
+  Shows the help message.
+
+- **`-i INPUT_PATH, --input-path INPUT_PATH`**
+
+  A path to the input file that you want to read the  `/Amazon/AzCore/Bootstrap/project_path` key from. If not supplied, then this command uses the input file `</USER_PATH>/.o3de/Registry/bootstrap.setreg` instead.
+
+
+<!-------------------------------------------------------------->
+
+## `set-global-project`
+
+Sets the specified project as the engine's global project. By default this is set in `C:\Users\USER_PATH\.o3de\Registry\bootstrap.setreg`. You can also specify a file path using `-o`.
+
+If you set a global project, then O3DE tools (such as the Asset Processor and Editor) will use the global project. You can override the global project by specifying a project with the `project-path` parameter when you launch the O3DE tools from the command line.
```

Comment:
```suggestion
If you set a global project, then O3DE tools (such as **Asset Processor** and **O3DE Editor**) use the global project. To override the global project, specify a project with the `project-path` parameter when you launch the O3DE tools from the command line.
```
* For the tool names, used **bold** and dropped the definite article for their proper naming (per our [terminology guidelines](https://docs.o3de.org/docs/contributing/to-docs/terminology/).)
* Minor edits for tense (present preferred) and flow.